### DISCUSSION TITLE
material Color Picker in Swing - possible?

### DISCUSSION THREAD

**Discussion by zanderson9** [vincenzopalazzo/material-ui-swing#162] [Ideas]
Hi Vincent @vincenzopalazzo,

This issue is an inquiry/enhancement, as follows: 

JMARS is very rich in various user dialogs. One of those dialogs that is used widely throughout the application is Color Picker. We are using Swing-style Color Picker (attached), which would be great to replace with ["material" Color Picker](https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=CE93D8).
Furthermore, our UX Design calls for the material look (see attached).

Is it possible to implement material Color Picker in material-ui-swing library?

**Swing Color Picker, currently in use by JAMRS**
![SwingColorPicker](https://user-images.githubusercontent.com/49419742/56988253-71a38200-6b44-11e9-9e29-2f176bc0ad12.png)

**UX Design calls for material look**:
![ColorPicker](https://user-images.githubusercontent.com/49419742/56988325-9ac41280-6b44-11e9-885c-e53feef15125.png)

**Comment by vincenzopalazzo** [comment]
Hi @zanderson9,

This is the effect your desired, right? 
![Selection_002](https://user-images.githubusercontent.com/17150045/56991461-008dba00-6b98-11e9-8c5f-e2f92a120c9b.png)

**Comment by zanderson9** [comment]
Yes!!

**Comment by vincenzopalazzo** [comment]
why not try, I'm adding the your suggestion, but I'm not adding the fix in the version 1.0 because the component UI must have re-written

**Comment by zanderson9** [comment]
I agree.  I meant exactly this, see if we can **_try_** and implement material Color Picker, as it is _widely_ used in JMARS. And then, who knows, this implementation may create a base for expanding to implement other [material Pickers](https://material.io/search.html?q=picker), which are in popular demand!

**Comment by vincenzopalazzo** [comment]
Let's work on :)
